tools/scoring/join_results.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
+
+if [[ $# -lt 2 ]]; then
+    echo "Usage: $0 base-result.csv [adjunct-result.csv [...]]"
+    exit 1
+fi
+
+bname=$(csvcut $1 -c 2 |tail +2 |head -1 |cut -d '/' -f 1)
+
+(
+cat $1;
+shift
+while [[ $# -ne 0 ]]; do
+    adjname=$(csvcut $1 -c 2|tail +2 |head -1 |cut -d '/' -f1)
+    cat $1 |tail +2 |while read -r line; do
+        echo ${line/${adjname}/${bname}};
+    done
+    shift
+done
+) |csvsort -c 2 -y 0 --no-inference
```

Comment:
If `csvsort` is too slow, we may also consider something like:

```
LANG=C
head -1 $1;
(
tail +2 $1;
....
) | sort -k2 -t','
```

Reply:
I think it's now ok.  Without the options it was super slow, but now the run time is minor compared to computing scores.